plugins/chat/assets/javascripts/discourse/components/modal/chat/create-channel.hbs

```diff
@@ -0,0 +1,137 @@
+        <label for="description" class="modal-chat-create-channel__label">
+          {{i18n "chat.create_channel.description"}}
+        </label>
+        <Input
+          name="description"
+          class="modal-chat-create-channel__input"
+          @type="textarea"
+          @value={{this.description}}
+        />
+      </div>
+
+      <div class="modal-chat-create-channel__control">
+        <label class="modal-chat-create-channel__label">
+          {{i18n "chat.create_channel.choose_category.label"}}
+        </label>
+        <CategoryChooser
+          @value={{this.categoryId}}
+          @onChange={{action this.onCategoryChange}}
+          @options={{hash
+            formName="chatable_id"
+            none="chat.create_channel.choose_category.none"
+          }}
+        />
+
+        {{#if this.categoryPermissionsHint}}
+          <div
+            class={{concat-class
+              "modal-chat-create-channel__hint"
+              (if this.loadingPermissionHint "loading-permissions")
+            }}
+          >
+            {{this.categoryPermissionsHint}}
+          </div>
+        {{/if}}
+      </div>
+
+      {{#if this.autoJoinAvailable}}
+        <div class="modal-chat-create-channel__control -auto-join">
+          <label class="modal-chat-create-channel__label">
+            <Input
+              name="auto_join_users"
+              @type="checkbox"
+              @checked={{this.autoJoinUsers}}
+            />
+            <div class="auto-join-channel">
+              <span class="modal-chat-create-channel__label-title">
+                {{i18n "chat.settings.auto_join_users_label"}}
+              </span>
+              <p class="modal-chat-create-channel__label-description">
+                {{#if this.categoryName}}
+                  {{i18n
+                    "chat.settings.auto_join_users_info"
+                    category=this.categoryName
+                  }}
+                {{else}}
+                  {{i18n "chat.settings.auto_join_users_info_no_category"}}
+                {{/if}}
+              </p>
+            </div>
+          </label>
+        </div>
+      {{/if}}
+
+      {{#if this.threadingAvailable}}
+        <div class="modal-chat-create-channel__control -threading-toggle">
+          <label class="modal-chat-create-channel__label">
+            <Input
+              name="threading_enabled"
+              @type="checkbox"
+              @checked={{this.threadingEnabled}}
+            />
+            <div class="threading-channel">
+              <span class="modal-chat-create-channel__label-title">
+                {{i18n "chat.create_channel.threading.label"}}
+              </span>
+              <p class="modal-chat-create-channel__label-description">
+                {{i18n "chat.settings.channel_threading_description"}}
+              </p>
+            </div>
+          </label>
+        </div>
+      {{/if}}
+
+      <div class="modal-footer">
```

Comment:
Can now be updated to use https://github.com/discourse/discourse/commit/9a1a3906c4dad3e09b6f0515ccedc78d7bda8bdb